elfi/testbench/testbench.py

```diff
@@ -0,0 +1,295 @@
+"""This module implements testbench-functionality in elfi."""
+
+import logging
+
+import numpy as np
+
+from elfi.visualization.visualization import ProgressBar
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['Testbench', 'TestbenchMethod']
+
+
+class Testbench:
+    """Base class for comparing the performance of LFI-methods.
+
+       One elfi.Model can be inferred `repetitions`-times with
+       each of the methods included in `method_list`.
+
+    Attributes
+    ----------
+    model : elfi.Model
+        elfi.Model which is inferred.
+    method_list : list
+        List of elfi-infernce methods.
+    repetitions : int
+        How many repetitions of models is included in the testbench.
+    seed : int, optional
+
+
+    """
+
+    def __init__(self,
+                 model=None,
+                 repetitions=1,
+                 observations=None,
+                 reference_parameter=None,
+                 reference_posterior=None,
+                 progress_bar=True,
+                 seed=None):
+        """Construct the testbench object.
+
+        Parameters
+        ----------
+        model : elfi.Model
+            elfi.Model which is inferred. Needs to have get_model-method.
```

Comment:
not sure the information here is correct since this current version at least approves as input an `elfi.ElfiModel` which does not have a get_model-method

Reply:
This was a legacy artifact from earlier commit. Modified.